include/helper_functions.php

```diff
@@ -1,10 +1,45 @@
 <?php
     
-    // Function that formats output, easier to read than var_dump()
+    // function that formats output, easier to read than var_dump()
     function debugOutput($array){
 
         $clean = htmlspecialchars( print_r( $array, true ) );
 
         echo"<pre>".$clean."</pre>";
-      
+
+    }
+
+    // function to accept CORS requests from the Chrome extension
+    function acceptCorsRequestsFromTheChromeExtension() {
+
+    // Allow from any origin
```

Comment:
I think we wrote this together, so maybe this is on me, but this function has incorrect indentation.